src/pages/gateway/batching.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Batching with API Mesh
+description: Learn how API Mesh uses batching. 
+kewords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Batching with API Mesh for Adobe Developer App Builder
+
+Batching allows you to combine a group of requests into a single request, turning multiple queries into a single one. Compared to sending multiple queries simultaneously, batched requests result in better response times. They also avoid issues with rate-limiting.
+
+Consider a scenario where you are using the following mesh, where `<reviews_api>` is a third-party API that contains reviews for your products by SKU. Each review has a `review`, `customer_name`, and `rating` field.
+
+```json
+{
+    "meshConfig": {
+        "sources": [
+            {
+                "name": "Products",
+                "handler": {
+                    "graphql": {
+                        "endpoint": "https://venia.magento.com/graphql"
+                    }
+                }
+            },
+            {
+                "name": "Reviews API",
+                "handler": {
+                    "graphql": {
+                        "endpoint": "<Reviews_API_URL>",
+                        "useGETForQueries":true
+                    }
+                }
+            }
+        ]
```

Comment:
@jhadobe : We will need to update the mesh to add the additionalTypeDefs and additionalResolvers for this non-batching example. Currently, the linkage between two sources based on product reviews is not shown. Let me know if we have purposefully left the configuration.

Reply:
@simrangulati this is the base scenario, before configuring batching.

Reply:
I do not believe it is necessary. I can query the non-batching query against this example mesh and it produces the expected response.

Reply:
@jhadobe : The query we are using for this example should be same as the last query so we need to update the mesh

Reply:
@simrangulati I don't think so. This mesh uses the unbatched query to show the user the difference between batching and not batching.